test/parallel/test-cli-syntax.js

```diff
@@ -140,3 +140,26 @@ syntaxArgs.forEach(function(args) {
     }));
   });
 });
+
+// should work with -r flags
+['-c', '--check'].forEach(function(checkFlag) {
+  ['-r', '--require'].forEach(function(requireFlag) {
+    const preloadFile = fixtures.path('no-wrapper.js');
+    const file = fixtures.path('syntax/illegal_if_not_wrapped.js');
```

Comment:
`fixtures.path('syntax', 'illegal_if_not_wrapped.js')`

Reply:
@richardlau Updated. You might do a follow-up PR to tweak the rest of the files usage if that is important to you.